packages/contracts/contracts/implementations/SdeBitmapsCore.sol

```diff
@@ -53,7 +72,7 @@ library SdeBitmapsCore {
    */
   function setBit(BitMap storage bitmap, uint256 bitIndex) internal {
     uint256 bucket = bitIndex >> 8;
```

Comment:
Sir I need you to use getBucketFromBitIndex

Reply:
I took this opportunity to condense some of the boiler-plate which should also reduce stack memory usage too.